packages/authentication/modules/access/useLogout/types.d.ts

```diff
@@ -0,0 +1,3 @@
+import { ICookieName } from '../../../types/auth'
+
+export interface IUseLogout extends ICookieName {}
```

Comment:
nothing else to add here? is it just for using another name?

Reply:
yee, that one only receives `cookieName` as prop right now, I kept the interface name just in case we wanna add more into it in the future